.buildkite/scripts/pipelines/security_solution_quality_gate/create_docker_image.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+if [ -z "${KIBANA_LATEST+x}" ] || [ "$KIBANA_LATEST" = "0" ]; then
```

Comment:
Why copy instead of sourcing .buildkite/scripts/steps/artifacts/docker_image.sh?

Reply:
The reason that I copied it is the fact that every time I run this pipeline (every six hours approximately) it would also go through the gpctl promotion for the kibana-controller. Moreover the steps to build dependencies and Upload CDN assets will run. 

If you agree to go through all these every time I run this pipeline then I am more than happy to reuse the existing script. To be fair I was literally thinking to do it, but then I thought that promoting the image everytime would be a complication.

Reply:
This is a good reason not to use the whole `docker_image.sh` in this case, but the reverse could be applied.

If you can separate the responsibility of checking, creating and  docker image and uploading it to a file, the original `docker_image.sh` could use it before initiating a gpctl promotion, or going through with the rest of the actions in the file, and we'd avoid this obvious duplication

Reply:
@jbudz suggested to even rename the uploaded image file to reflect the secutity solution quality gate.

In case we need to not duplicate at all, @delanni What you suggest is actually a very good idea but I would suggest to either be handled by your team and let me integrate it once is done, OR let me know to add it in my queue as a ticket and do it in a separate PR.